07-regression1.Rmd

```diff
@@ -301,7 +315,8 @@ Here we can see that the smallest RMSPE occurs when $K =$ `r sacr_min %>% pull(n
 
```

Comment:
We should plot K versus RMSPE as we do for K vs accuracy in the previous chapter. Learners have to invert the values to tuning RMSPE from accuracy (looking for minimum RMSPE compared to maximum accuracy), and graphically seeing the relationship between K & RMSPE would help reinforce that.

Reply:
Also, the table formatting of `sacr_results` is a bit off in the book.

Reply:
Also, as suggested in the classification chapters, we should explain the output of `sacr_results` better. What are all the columns? Also, what is `rsq`?

Reply:
handled all of this